usmqe/api/tendrlapi/user.py

```diff
@@ -0,0 +1,142 @@
+        """ Edit a single user
+
+        Args:
+            username: name of user that is going to be updated
+            data: dictionary data about user
+                  have to contain: email, username, name, role
+            asserts_in: assert values for this call and this method
+        """
+        pattern = "users/{}".format(username)
+        request = requests.put(
+            pytest.config.getini("usm_api_url") + pattern,
+            json.dumps(data),
+            auth=self._auth)
+        self.print_req_info(request)
+        self.check_response(request, asserts_in)
+# TODO check json comparision
+        return request.json(encoding='unicode')
+
+    def user_add(self, user_in, asserts_in=None):
+        """ Add user throught **users**.
+
+        Name:        "POST_users",
+        Method:      "POST",
+        Pattern:     "users",
+
+        Args:
+            user_in: dictionary info about user
+                     have to contain: name, username, email, role, password, password_confirmation
+            asserts_in: assert values for this call and this method
+        """
+        pattern = "users"
+        request = requests.post(
+            pytest.config.getini("usm_api_url") + pattern,
+            data=json.dumps(user_in),
+            auth=self._auth)
+        self.print_req_info(request)
+        self.check_response(request, asserts_in)
+        user_in.pop("password", None)
+        user_in.pop("password_confirmation", None)
+        stored_user = self.user(user_in["username"])
+        pytest.check(
+            user_in == stored_user,
+            """Information sent: {}, information stored in database: {},
+            These should match""".format(user_in, stored_user))
+        return stored_user
+
+    def user(self, username, asserts_in=None):
+        """ Get user info..
+
+        Name:        "GET_user",
+        Method:      "GET",
+        Pattern:     "users/{username}",
+
+        Args:
+            username: name of user stored in database
+            asserts_in: assert values for this call and this method
+        """
+        pattern = "users/{}".format(username)
+        request = requests.get(
+            pytest.config.getini("usm_api_url") + pattern,
+            auth=self._auth)
+        self.print_req_info(request)
+        self.check_response(request, asserts_in)
+
+        return request.json(encoding='unicode')
+
+    def check_user(self, user_data, asserts_in=None):
+        """ Check if there is stored user with given attributes.
+
+        Args:
+            user_data: user data that are going to be checked
+            asserts_in: assert values for this call and this method
+        """
+        stored_data = self.user(user_data["username"], asserts_in=asserts_in)
+        pytest.check(
+            stored_data == user_data,
+            "Json of stored user: {0}\n\tAnd of checked one: {1}\n\tShould be equal."
+            .format(stored_data, user_data))
```

Comment:
Could you reformat this a bit? I don't like the `.format(...)` on the beginning of the line - I'm surprised it's even possible :-). For example, you could save the sting into `msg` variable instead.